app/models/action.rb

```diff
@@ -1,6 +1,10 @@
 class Action < ActiveRecord::Base
   belongs_to :page
   belongs_to :action_user
+  after_create :update_page_action_count
 
+  def update_page_action_count
+    self.page.action_count = self.page.actions.count
```

Comment:
> Why not just increment by 1? 

My understanding is that the code that handles actions won't count duplicate actions by the same person. IE, if I sign a page three times, I won't get an action added to the actions table three times. So, using `page.actions.count` on create would shortcut the issue where I'm signing a petition for the third or fourth time and all those get counted as signatures per the thermometer but aren't actually signatures in AK or any real sense of the term.